docs/sources/structure/topic-types/task/index.md

```diff
@@ -38,7 +38,39 @@ A _task_ topic includes the following elements:
 
   If there are no prerequisites, don't include this section.
 
-- **Steps heading**: The steps heading indicates that the steps are about to begin.
+- **Steps heading**: Optional. The steps heading indicates that the steps are about to begin. Often follows "Before you begin" to delineate when the requirements end and the task starts. The steps heading may say "Steps" or a succinct description of a task.
```

Comment:
These guidelines were written when we thought most of our documents would document one task per Markdown file. In those cases, we do want the Steps heading. 

Then, @knylander-grafana (I think) put together the multiple tasks topic template https://github.com/grafana/writers-toolkit/blob/main/docs/static/templates/multiple-tasks-template.md , which doesn't have any corresponding guidelines. Maybe that's fine? I'd say a majority of our writing follows the multiple tasks per topic approach, in which case we'd want the writer to know to include the Goals of procedure bit at the top and to ensure each task within the topic doesn't have intro text beyond the stem sentence.

Reply:
May be should incorporate the multi-task example into these instructions? What do you think?

Reply:
what changes do we need to make to this PR so we can get it merged?

Reply:
Just realized this was still open. I'll get the updates in and we can get it done.

Reply:
Perhaps we can leave that for a follow up PR and get this merged? Or is the information for multi-task topics important to this revised version of the task topic documentation?

Reply:
Keeping this on my radar. Can we merge this and follow up later?